src/i18n/i18n.csv

```diff
@@ -59,7 +59,7 @@ Archived Web page template,arch-pg,This Web page has been archived on the Web.,C
 Tabbed interface,tab-rot-off,Stop tab rotation,Arrêter la rotation d\'onglets,Stop tab rotasie,Stop rotacionin tab,ايقاف الدوران علامة التبويب,Կանգ էջանիշը պտույտը,Спри да се върти раздела,旋轉鍵-停止,停止标签旋转,Zastavit kartě otáčení,Stop de tabrotatie,Seiska sakkide rotatsioon,Stoppen Registerkarte Drehung,Σταματήστε να περιστροφή καρτέλα,टैब रोटेशन रोकें,Tabulálás engedélyezése,Stöðva flipann snúningur,Hentikan rotasi tab,Arresto rotazione scheda,ᓱᑯᑦᑎᐊᓅᖅᑕᐅᑦ ᑲᐃᕙᕐᓂᖓ ᓄᖅᑲᕐᓕ,タブローテーションを停止,탭 회전을 중지,Pārtraukt rotāciju,Sustabdyti rotaciją,Zatrzymaj przewijanie zakładek,Interromper a rotação das abas,Interromper a rotação das abas,Остановить ротацию вкладки,Zastaviť kartu otáčania,Detener rotación de pestañas,หยุดหมุนแท็บ,Tab dönüşünü durdur,Зупинити ротацію вкладки,Ngừng xoay tab
 ,tab-rot-on,Start tab rotation,Lancer la rotation d\'onglets,Begin blad rotasie,Fillojnë rrotullimin tab,بدء دوران التبويب,Սկիզբ էջանիշը պտույտը,Започнете въртене раздела,旋轉鍵-啟動,开始标签旋转,Zahájit kartě otáčení,Start de tabrotatie,Alusta sakkide rotatsiooni,Starten Registerkarte Drehung,Ξεκινήστε το περιστροφή καρτέλα,टैब रोटेशन प्रारंभ,Tabulálás kikapcsolása,Byrja flipann snúningur,Mulai rotasi tab,Inizio rotazione scheda,ᓱᑯᑦᑎᐊᓅᖅᑕᐅᑦ ᑲᐃᕙᕐᓂᖓ ᐱᒋᐊᕐᓕ,タブローテーションを再生,탭 회전을 시작,Sākt rotāciju,Pradėti rotaciją,Rozpocznij przewijanie zakładek,Iniciar a rotação das abas,Iniciar a rotação das abas,Начать ротацию вкладки,Dátum kartu otáčania,Comenzar rotación de pestañas,เริ่มหมุนแท็บ,Tab dönüşünü başlat,Почати ротацію вкладки,Bắt đầu luân chuyển tab
-,tab,Tab #num#,Onglet #num#,Tab #num#,Tab #num#,#num# التبويب,ականջակալ #num#,Раздел #num#,标签 #num#,标签 #num#,Tab #num#,Lab #num#,Sakk #num#,Tab #num#,Αυτί #num#,टैब #num#,Tab #num#,Tab #num#,Tab #num#,Scheda #num#,ᓱᑯᑦᑎᐊᓅᖅᑕᐅᑦ #ᓈᓴᐅᑦ#,タブ #num#,탭 #num#,Tab #num#,Skirtukas #num#,Patka #num#,Aba #num#,Aba #num#,Вкладка #num#,Tab #num#,Pestaña #num#,แท็บ #num#,Sekme #num#,табуляція #num#,Chuyển hướng #num#
+,tab,Tab #num#,Onglet #num#,Tab #num#,Tab #num#,#num# التبويب,ականջակալ #num#,Раздел #num#,标签 #num#,标签 #num#,Tab #num#,Lab #num#,Sakk #num#,Tab #num#,Αυτί #num#,टैब #num#,Füllemez #num#,Tab #num#,Tab #num#,Scheda #num#,ᓱᑯᑦᑎᐊᓅᖅᑕᐅᑦ #ᓈᓴᐅᑦ#,タブ #num#,탭 #num#,Tab #num#,Skirtukas #num#,Patka #num#,Aba #num#,Aba #num#,Вкладка #num#,Tab #num#,Pestaña #num#,แท็บ #num#,Sekme #num#,табуляція #num#,Chuyển hướng #num#
```

Comment:
Please revert this change

"Füllemez" to "Tab" car "Füllemez" veux dire "plaque d'oreille" ce qui n'est une traduction exacte dans ce contexte-ci.